tsdb/engine/tsm1/wal_test.go

```diff
@@ -6,11 +6,13 @@ import (
 	"io/ioutil"
 	"os"
 	"reflect"
+	"sort"
 	"testing"
 
 	"github.com/golang/snappy"
 	"github.com/influxdata/influxdb/v2/pkg/slices"
 	"github.com/influxdata/influxdb/v2/tsdb/engine/tsm1"
+	"github.com/stretchr/testify/assert"
```

Comment:
```suggestion
	"github.com/stretchr/testify/require"
```
IMO makes the testing simpler, since it bails at the first mismatched expectation